var/spack/repos/builtin/packages/plink/package.py

```diff
@@ -32,10 +32,21 @@ class Plink(Package):
 
     homepage = "https://www.cog-genomics.org/plink/1.9/"
 
+    version('1.9-beta5', '737545504ae19348a44a05fa69b75c28',
+            url='https://github.com/chrchang/plink-ng/archive/b15c19f.tar.gz')
```

Comment:
We don't generally provide beta versions of software by default. Can you add `preferred=True` to 1.9 so that it remains the default version that Spack installs?

Reply:
It seems that the url for version 1.9 cannot be fetched now. And I looked at the homepage for plink 1.9, and it provides binary version of 1.9 beta 5.4 now.
See https://www.cog-genomics.org/plink/1.9/

Reply:
It looks like 1.9 hasn't even been released yet. I would be more comfortable with adding 1.07 and setting that to be the default:
http://zzz.bwh.harvard.edu/plink/download.shtml